README.md

```diff
@@ -2,6 +2,67 @@
 
 These are the scripts I use while releasing [The Foreman](https://theforeman.org). They automate most of the [release process](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb).
 
+## Setup
+
+Required dependencies:
+
+ * gopass
+ * python3
+  * click
```

Comment:
the indentation is still off by one ;)

Reply:
Ohhh that was intentional! I was trying to identify "python3" required packages since something like `click` is ambiguous if thats an RPM or python or hell some other project all together.